7.0/config/vhost/slim-ssl.conf

```diff
@@ -0,0 +1,16 @@
+<VirtualHost *:443>
+    DocumentRoot /var/www/html/public
+    UseCanonicalName on
+
+    SSLEngine on
+    SSLCertificateFile /var/www/html/ssl/local.crt
+    SSLCertificateKeyFile /var/www/html/ssl/local.key
+    SSLCertificateChainFile /var/www/html/ssl/local.crt
```

Comment:
I think we can drop this (from all configs) as long as we're using a new enough Apache. New behaviour is analogous to nginx's.

> SSLCertificateChainFile is deprecated

> SSLCertificateChainFile became obsolete with version 2.4.8, when SSLCertificateFile was extended to also load intermediate CA certificates from the server certificate file.

https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile